x/wasm/keeper/querier.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 // DefaultGasCostBuildAddress is the SDK gas cost to build a contract address
-const DefaultGasCostBuildAddress = 5
+const DefaultGasCostBuildAddress = 500_000
```

Comment:
This is a Cosmos SDK gas value where 100_000-200_00 is the gas range of a full transaction with verification, state changes, events etc. I think we need some much smaller value here.


What about 10?

Reply:
Sounds good! I will update the pr 👍